napari/__init__.pyi

```diff
@@ -1,24 +1,34 @@
-import napari._qt.qt_event_loop
-import napari.plugins.io
 import napari.utils.notifications
-import napari.view_layers
-import napari.viewer
+from napari._qt.qt_event_loop import gui_qt as gui_qt
+from napari._qt.qt_event_loop import run as run
+from napari.plugins.io import save_layers as save_layers
+from napari.view_layers import (
+    view_image as view_image,
+)
```

Comment:
```suggestion
from napari.view_layers import view_image
```

??? I don't understand this at all

Reply:
(Sorry, I missed the above links because they related to a different kind of change. But I still don't like this...)

Reply:
Ok. The currently this syntax is enforced by "PYI015" https://beta.ruff.rs/docs/rules/#flake8-pyi-pyi rule of ruff. 
Unfortunately, this rule does not accept `__all__` even when mypy accepts it. 

But the current code was wrong as it does not contain explicit reexport or `__all__`.

I'm open to disabling "PYI015". But I also see profit from enforcing this rule. It will be nice to hear from other core devs (I will merge this PR on Sunday, but if there is no feedback immediately will open a followup).

Reply:
> But I also see profit from enforcing this rule.

Such as...? 😂 I prefer `__all__` by a significant margin... As in my other reviews... Code should be obvious in what it does... I suspect I would not be the first person to be totally baffled by these lines. I also think that we should combine the import lines so that we don't have so many

```
from X import (
    Y as Y
)
```

It's really quite horrible imho!

Reply:
Multiple import as comes from ruff configuration https://beta.ruff.rs/docs/settings/#combine-as-imports

Reply:
I have combined imports. I will merge in about 12h to be before the automatic update. Then I will open the Issue for a longer discussion.